src/containers/AccessPointDetails/components/Status/index.js

```diff
@@ -8,7 +8,7 @@ import styles from '../../index.module.scss';
 
 const { Item } = Form;
 
-const Status = ({ data }) => {
+const Status = ({ data, isAlarmsEnabled }) => {
```

Comment:
change the name to `showAlarms` it's not descriptive. and make the default prop 'true'